cli/bpmetadata/path.go

```diff
@@ -0,0 +1,85 @@
+package bpmetadata
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+)
+
+const (
+	regexEx  = ".*/(examples/.*)"
+	regexMod = ".*/(modules/.*)"
+)
+
+func isPathValid(path string) (bool, error) {
+	_, err := os.ReadFile(path)
```

Comment:
Since we don't read any data, we can skip the read and just do a os.Stat.

Reply:
ack